tests/test_paperqa.py

```diff
@@ -3164,6 +3207,7 @@ async def test_reader_config_propagation(stub_data_dir: Path, multimodal: bool)
     ("filename", "query"),
     [
         ("dummy.docx", "What is the RAG system?"),
+        ("dummy_jap.docx", "What is the RAG system?"),
```

Comment:
Can you 'find' for `dummy.docx` in the tests, there's a few in `test_agents.py`. Just update the expected counts. We can probably generalize the assertions better, but for now let's just update